File: monobuild/tests/get_state.pl
#!/usr/bin/perl


use FindBin;
use lib "$FindBin::RealBin/..";

use Mono::Build;

my $platform = 'suse-93-i586';
my $package = 'gecko-sharp-2.0';
my $rev = 'r4543';

my $state = Mono::Build::getState($platform, $package, $rev);

print "State: $state\n";

File: pyutils/tests/get_state.py
#!/usr/bin/env python

import sys

sys.path += [ '../../pyutils' ]

import build
import datastore

platform = 'suse-93-i586'
package = 'gecko-sharp-2.0'
version = '4543'

build_info = datastore.build_info('HEAD', platform, package, version)

print build_info.get_state()


